src/Controller.res

```diff
@@ -1,10 +1,59 @@
+module OnChangeArg: {
+  @@ocaml.doc("
+This module is used to build event or value argument to pass down to the `onChange` callback.
+")
+
+  type rec t
+
+  @ocaml.doc("Takes a callback that accepts a `ReactEvent.Form.t`.
+Useful for dealing with native form inputs.")
+  let event: ReactEvent.Form.t => t
+
+  @ocaml.doc("Takes a callback that accepts a `Js.Json.t`.
+This function allows you to set the input value to any arbitrary value.")
+  let value: Js.Json.t => t
+} = {
+  @unboxed
+  type rec t = Any('a): t
+
+  let event = eventHandler => Any(eventHandler)
+
+  let value = valueHandler => Any(valueHandler)
+}
+
+@ocaml.doc("Simple helper that simplifies `onChange` argument handling when it's passed down to native inputs.
+
+_Has a small runtime cost._
+
+Example:
+
+```
+<Controller
+  render={({field: {onChange}}) =>
+    <input onChange={event => onChange(Controller.OnChangeArg.event(event))} />
+  }
+/>
+```
+
+Can be written:
+
+```
+<Controller
+  render={({field: {onChange}}) =>
+    <input onChange={Controller.handleEvent(onChange)} />
+  }
+/>
+```
+")
+let handleEvent = (onChange, event) => onChange(OnChangeArg.event(event))
```

Comment:
To me this one is really not necessary and kinda defy the original goal: to be runtime free.

Though, the runtime cost being very small I think we can keep it 😄 

What do you guys think @vtrofin @infothien ?

Reply:
For me I prefer to just write 
```
<Controller
  render={({field: {onChange}}) =>
    <input onChange={event => onChange(Controller.OnChangeArg.event(event))} />
  }
/>
```
=)))

Reply:
Me too actually, i added this one as a compromise just in case XD

If @vtrofin is ok as well, let's drop this function :+1:

Reply:
@infothien Actually I miss the good old days of ` onChange={onChange << Controller.OnChangeArg.event}` that was much, much clearer 😞

Reply:
thank you @gaku-sei and @infothien for pointing this out. I'm onboard with the change as well